IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcBuiltElementType/Documentation.md

```diff
@@ -7,4 +7,4 @@ The _IfcBuiltElementType_ can be instantiated in the case when arbitrary built e
 
 Occurrences of subtypes of the _IfcBuiltElementType_ are represented by instances of the appropriate subtypes of _IfcBuiltElement_.  
 
-> HISTORY&nbsp; HISTORY New entity in IFC2x2.
\ No newline at end of file
+> HISTORY&nbsp; HISTORY New entity in IFC2x2.
```

Comment:
Is this intentional, that HISTORY is stated twice?

Reply:
Thanks. I don't think it is supposed to be like that. I have corrected it.